specification/peering/resource-manager/Microsoft.Peering/stable/2021-01-01/peering.json

```diff
@@ -3398,11 +3405,19 @@
       "type": "object",
       "properties": {
         "peeringServiceLocation": {
-          "description": "The PeeringServiceLocation of the Customer.",
+          "description": "The location (state/province) of the customer.",
+          "type": "string"
+        },
+        "providerPrimaryPeeringLocation": {
```

Comment:
By adding these parameters in the middle it will result in a breaking change.  Can we add them to the end of the list?